tests/stateful_consumer/test_bootstrap_state.py

```diff
@@ -114,7 +114,7 @@ def test_snapshot_loaded(self, create_consumer) -> None:
         create_consumer.return_value = kafka_consumer
 
         bootstrap = BootstrapState(
-            "cdc_control", "somewhere", "something", get_dataset("groupedmessage"),
+            "cdc_control", "somewhere", "something", get_storage("groupedmessages"),
```

Comment:
This is not going to type check. `get_storage` returns a Storage while the BootstrapState constructor accepts a CdcStorage. While we are not as strict with tests as we are with production code, this is an easy fix.

Reply:
Oops, thanks